### PR TITLE
docs: Provide alternative login option to Azure CLI

### PR DESCRIPTION
**Pull request by aliphys**
I have had difficulty logging into Azure via the `az login` PowerShell command. Using the option `--use-device-code` was smooth and easy.

Adding this to the CLI install section to share this knowledge with others (and my future self). 😁

This PR includes the commit only for this issue and replaces https://github.com/MicrosoftDocs/azure-ai-docs/pull/61

### CODE REVIEW

**Comment by aliphys** [comment]
#sign-off

**Comment by aliphys** [comment]
#sign-off

**Comment by Jak-MS** [comment]
@sdgilley 
Can you review the proposed changes? 

**Important**: When the changes are ready for publication, adding a `#sign-off` comment is the best way to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by aliphys** [comment]
#sign-off

File: articles/ai-studio/includes/install-cli.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 title: include file
 description: include file
-author: sdgilley
+author: sdgilley, aliphys
```
Comment (Jak-MS) [inline articles/ai-studio/includes/install-cli.md]:
```suggestion
author: sdgilley
```

**Comment by aliphys** [comment]
Could we proceed with merging this PR if no further changes are needed? If there are any additional changes or feedback required, please let me know. Thank you!

**Comment by sdgilley** [comment]
#sign-off

**Pull request merged**